gdshelpers/helpers/small.py

```diff
@@ -47,6 +48,20 @@ def id_to_alphanumeric(column, row):
     return int_to_alphabet(row) + str(int(column))
 
 
+def parse_alphanumeric(text):
+    """
+    Do the reverse of `id_to_alphanumeric`.
+    Returns the (column, row) tuple.
+    """
+    regex = "([A-Z]*)[^0-9]*([0-9]*)"
+    m = re.match(regex, text.strip(), re.I)
```

Comment:
removing whitespaces would be also something not expected by the description - I'd say the user should do this if necessary, what do you think?
```suggestion
    m = re.match(regex, text, re.I)
```